src/features/canvass/components/LocationDialog/pages/HouseholdsPage.tsx

```diff
@@ -42,11 +46,16 @@ const HouseholdsPage: FC<Props> = ({
   onClose,
   onCreateHousehold,
   onSelectHousehold,
+  onStartHouseholdsVisit,
   location,
+  resetSelection,
 }) => {
   const messages = useMessages(messageIds);
+  const theme = useTheme();
   const households = useHouseholds(location.organization_id, location.id);
   const [adding, setAdding] = useState(false);
+  const [reportMany, setReportMany] = useState(false);
+  const [selectedHouseholds, setSelectedHouseholds] = useState<number[]>([]);
```

Comment:
This looks to me like a duplication of the state from `LocationDialog/index.tsx`, which also has a `selectedHouseholdIds` state. If two components need the same state, the solution is to move the state up in the hierarchy. In this case, the state in `LocationDialog/index.ts` should be enough, by passing it in as props to this component.

The `reportMany` and `selectedHouseholds` in here can be derived from that state from the parent component.